client/boot/app.js

```diff
@@ -0,0 +1,131 @@
+// Initialize localStorage polyfill before any dependencies are loaded
+require( 'lib/local-storage' )();
+if ( process.env.NODE_ENV === 'development' ) {
+	require( 'lib/wrap-es6-functions' )();
+}
+
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+import page from 'page';
+import qs from 'querystring';
+import ReactClass from 'react/lib/ReactClass';
+import i18n, { setLocale } from 'i18n-calypso';
+import url from 'url';
+
+/**
+ * Internal dependencies
+ */
+import accessibleFocus from 'lib/accessible-focus';
+import createReduxStoreFromPersistedInitialState from 'state/initial-state';
```

Comment:
This `import` creates a dependency on every reducers in `state`. Would be cool to move it to `./project/xxx` so each "project" is responsible for the instantiating the reduxStore and the middlewares they need.